app.js

```diff
@@ -13,6 +13,28 @@ function hasFilter(name, filters) {
   return false;
 }
 
+function getParameterByName(name, url) {
+  if (!url) url = window.location.search;
+  name = name.replace(/[\[\]]/g, "\\$&");
+  var regex = new RegExp("[?&]" + name + "(=([^&#]*)|&|#|$)"),
+  results = regex.exec(url);
```

Comment:
nit: use `var results` (we don't use the var x, y; notation)
and add a new line after this one.